calendar@deeppradhan/files/calendar@deeppradhan/calendar.js

```diff
@@ -0,0 +1,25 @@
+// Names of the Month
+const MONTH_NAMES = ["January", "February", "March", "April", "May", "June",
```

Comment:
On Cinnamon 3.8, these constant variables will need to be changed to `var` because of changes to GJS. Any variables intended to be exported should use `var`.